### ISSUE TITLE
Change mail topic

### ISSUE THREAD

**Issue by 0xERR0R** [mcuadros/ofelia#125]
The mail topic looks currently like this:

`[Execution successful] Job job-name finished in 3m19.830133383s`
or
`[Execution failed] Job job-name finished in 1.768168ms`

There are 3 important values: status, job name and execution duration. But there is also to much "noise", it should be more concise.

Proposal:
- Status in upper case
- remove "Execution"
- duration, show only one digit after the collon

Example:
`[SUCCESSFUL] job-name finished in 3m19.8s`
or
`[FAILED] job-name finished in 1.78ms`

This would make it easier to filter the mails.

**Comment by Trane9991** [comment]
Well, this makes some sense, however changing the mail topic would break mail filtering for all the users who will update ofelia version, and since we added per commit images tagging just recently, I suspect that many people use ofelia with `latest` tag, so this could be quite unexpected for some users.

**Comment by 0xERR0R** [comment]
Good point. Maybe some kind of configuration possibility. It not configured - use default as now

**Comment by bereldhuin** [comment]
+1
